### ISSUE TITLE
Wrong Content-Transfer-Encoding on multipart messages

### ISSUE THREAD

**Issue by dpc22** [roundcube/roundcubemail#7170]
We upgraded to Roundcube 1.4.2 last week.

One of my users just reported that they received a bounce from a LISTSERV mailing list that they subscribe to:

```
Your message contains a QUOTED-PRINTABLE encoded composite MIME part, which is
not allowed by Internet standards (see  RFC2045, section 6.4, paragraph 5). In
plain English, an error in your mail program, or in a firewall or mail gateway
between your computer and LISTSERV, has caused  an error in the format of your
message.
```
RFC2045 , section 6.4, paragraph 5 states:

```
Certain Content-Transfer-Encoding values may only be used on certain
media types.  In particular, it is EXPRESSLY FORBIDDEN to use any
encodings other than "7bit", "8bit", or "binary" with any composite
media type, i.e. one that recursively includes other Content-Type fields.  
```

A brief test with a message which triggers quoted-printable encoding ("£££" in the message text is my normal approach to trigger quoted-printable) and an attachment does indeed seem to indicate a change in behaviour somewhere between Roundcube 1.0.12 (that we were previously running) and Roundcube 1.4.2.

I attach a test message that I generated (later updated to be copy from my sent folder):

[Test.eml.txt](https://github.com/roundcube/roundcubemail/files/4054959/Test.eml.txt)

The significant bit is:

```
Content-Type: multipart/mixed;
  boundary="=_7b1d300b2c34571f16e451985990c4b7"
Content-Transfer-Encoding: quoted-printable
```
where the "Content-Transfer-Encoding:" line is new.

I don't see this with Roundcube 1.0.12 running on the same machine.

**Comment by alecpl** [comment]
Confirmed. It is `8bit` with disabled Preferences > Composing Messages > Advanced options > Use MIME encoding for 8-bit characters, though.

**Comment by alecpl** [comment]
Fixed with https://github.com/pear/Mail_Mime/commit/7721b301ff9fcbcaa66. I'll close the ticket after a new version of Mail_mime is available.

**Comment by alecpl** [comment]
Fixed in Mail_Mime 1.10.5.
